keras_cv/models/object_detection/yolo_v8/yolo_v8_layers.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from keras import layers
+
+BATCH_NORM_EPSILON = 1e-3
+BATCH_NORM_MOMENTUM = 0.97
+
+
+def apply_conv_bn(
+    inputs,
+    output_channel,
+    kernel_size=1,
+    strides=1,
+    activation="swish",
+    name=None,
+):
+    if kernel_size > 1:
+        inputs = layers.ZeroPadding2D(
+            padding=kernel_size // 2, name=f"{name}_pad"
+        )(inputs)
+
+    x = layers.Conv2D(
+        filters=output_channel,
+        kernel_size=kernel_size,
+        strides=strides,
+        padding="valid",
+        use_bias=False,
+        name=f"{name}_conv",
+    )(inputs)
+    x = layers.BatchNormalization(
+        momentum=BATCH_NORM_MOMENTUM,
+        epsilon=BATCH_NORM_EPSILON,
+        name=f"{name}_bn",
+    )(x)
+    x = layers.Activation(activation, name=name)(x)
+    return x
+
+
+def apply_csp_block(
```

Comment:
This method shouldn't exist once we migrate, because we should use the CSP darknet block instead.

I'm looking into switching to using the CSPDarkNet block now, even before the backbone is migrated.